gridstatus/ercot.py

```diff
@@ -1773,6 +1777,49 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+        if date == "latest":
+            doc = self._get_document(
+                report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+                date=date,
+                verbose=verbose,
+            )
+            df = pd.read_csv(doc.url, compression="zip")
+
+            return df
+        else:
+            doc = self._get_documents(
+                report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+                date=date,
+                verbose=verbose,
+                extension="csv",
+            )
+            df = pd.DataFrame()
+            for i in doc:
+                new_df = pd.read_csv(i.url, compression="zip")
```

Comment:
this appears to be missing handle of the date time parsing, dst changes, and setting of the time zone

for example: https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L2028

if you can modify parse_doc function you maybe be able to inherit some of this logic for free

Reply:
Oh i thought we were good to ignore that, i may just redo the rename_func logic to inherit that logic